### PR TITLE
Completed two-column-layout.

### PR DESCRIPTION
**Pull request by Alankim56**
2-column-layout

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  margin-bottom: 30px;
+  width: 100%;
+}
+
+h3 {
+  width: 60%;
+}
+
+p {
+  width: 70;
```
Comment (scrumptiousdev) [inline css-two-column-layout/styles.css:63]: This does not have any unit so it's not a valid CSS value

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+h2 {
+  font-size: 14px;
+  margin-bottom: 30px;
+  width: 100%;
+}
+
+h3 {
+  width: 60%;
```
Comment (scrumptiousdev) [inline css-two-column-layout/styles.css:59]: We shouldn't set width to h3 but rather depends text to fill the column

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+p {
+  width: 70;
+  font-size: 75%;
+  margin-left: 150px;
+  margin-right: 180px;
```
Comment (scrumptiousdev) [inline css-two-column-layout/styles.css:66]: We shouldn't use margins in order to position the texts

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  font-style: normal;
+  font-weight: 400;
+}
+
+.picture {
+  display: flex;
+  margin-left: 20px;
+  margin-right: auto;
+  width: 20%;
+  float: left;
```
Comment (scrumptiousdev) [inline css-two-column-layout/styles.css:34]: Float left is not needed on this exercise

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Within each column, you should rather utilize flex to create columns to separate image and text side by side

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Changes in progress?

**Comment by Alankim56** [comment]
css-two-column

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
What you want to do is to still use 1/2 columns. But within each 1/2 columns you can have a row and then columns within it

**Review by scrumptiousdev — APPROVED** [review]
Approved!

**Pull request merged**